cmd/fs-v1.go

```diff
@@ -118,14 +118,14 @@ func (fs fsObjects) Shutdown() error {
 	if err != errFileNotFound {
 		// A nil err means that multipart directory is not empty hence do not remove '.minio.sys' volume.
 		// A non nil err means that an unexpected error occured
-		return err
+		return toObjectErr(traceError(err))
```

Comment:
toObjectErr() will be useful only for s3 calls, ie it will be used to construct error response XML, as Shutdown() is an internal call we need not do toObjectErr(), but it is better to be consistent code wise i.e from the object layer we always return object-error not posix error. So doing toObjectErr() is fine.

Since we are returning trace info in the error now, you can delete the errorIf() statements in this function and instead do one errorIf() at the caller. (routers.go)